app/models/article.rb

```diff
@@ -0,0 +1,5 @@
+class Article < ApplicationRecord
+    validates :title, presence: true
+    validates :body, presence: true, length: { minimum: 10 }
```

Comment:
What does presence do?

Reply:
Presence validates that a title value and a body value contain at least one character and are not empty.